core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -132,7 +126,7 @@ public static boolean isBinary(int columnType) {
     }
 
     public static boolean isBoolean(int columnType) {
-        return columnType == ColumnType.BOOLEAN;
+        return columnType == BOOLEAN;
```

Comment:
we should compare tags here: ColumnType.tagOf(columnType) ==
this applies to other comparison fns